### ISSUE TITLE
Quickstart "when not to use" outdated with geomloss integration?

### ISSUE THREAD

**Issue by tmchartrand** [PythonOT/POT#678]
Unless I'm missing something, it seems the note on when not to use POT at https://pythonot.github.io/quickstart.html#when-not-to-use-pot is no longer relevant, since the geomloss functionality is now fully wrapped in POT? or is there some missing functionality not included?
Thanks!

**Comment by rflamary** [comment]
Yes that is true, we would appreciate it if you do a quick PR to update it maybe with a short example of how to activate geomloss.

We need a big revamp of the documentation with the New API also.

**Comment by rflamary** [comment]
This issue was fixed in PR #726
